app/services/paypal/refresh_identity.rb

```diff
@@ -0,0 +1,53 @@
+class Paypal::RefreshIdentity < BaseService
+  def initialize(publisher_id:)
+    @publisher = Publisher.find(publisher_id)
+    @paypal_connection = @publisher.paypal_connection
+  end
+
+  def perform
+    return if @paypal_connection.nil? || @paypal_connection.refresh_token.nil?
+    begin
+      access_token = fetch_access_token
+      raise_token_exception if access_token.nil?
+    rescue => e
+      require 'sentry-raven'
```

Comment:
You shouldn't need to require this because it's included during initialization of Rails 🌈 
```suggestion

```